.circleci/config.yml

```diff
@@ -1,14 +1,27 @@
 version: 2.1
 
-jobs:
-  hello:
+executors:
+  ruby:
     docker:
-      - image: alpine
+      - image: circleci/ruby:2.7.1
+
+jobs:
+  test:
+    executor: ruby
     steps:
-      - run: echo "Hello, world! 👋"
+      - checkout
+      - run: bundle install && bundle exec rspec
+  bintray_garbage_collection:
+    executor: ruby
+    steps:
+      - checkout
+      - run: bundle install && bundle exec ruby ./bin/bintray_garbage_collector.rb
```

Comment:
Two jobs: test and garbage collection. As you'll see below, `test` runs on every push (as long as there is a PR as per CircleCI default) while `bintray_garbage_collection` runs every day at midnight. According [to the docs](https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow) think "midnight" is relative to UTC

>  [`triggers`] enables you to schedule a workflow run by using cron syntax to represent Coordinated Universal Time (UTC) for specified branches.

and 

> **Note**: Scheduled workflows may be delayed by up to 15 minutes. This is done to maintain reliability during busy times such as 12:00am UTC. Scheduled workflows should not assume they are started with to-the-minute accuracy.